libs/shared/src/lib/sass/mixins/appearance/_mixins.scss

```
@@ -13,7 +13,7 @@ $hover: ':hover, .hover';
 $disabled: ':disabled, .disabled';
 $readonly: '.readonly';
 $checked: ':checked, .checked';
-$selected: '.selected, [aria-current]';
+$selected: '.selected, [aria-current], [aria-selected='true']';
```

```suggestion
$selected: '.selected, [aria-current]';
```
you are not using it because it will be on the host and you will add a class to `.base`